embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/CrashModule.kt

```diff
@@ -35,13 +38,8 @@ internal class CrashModuleImpl(
         CrashFileMarker(markerFile, initModule.logger)
     }
 
-    override val lastRunCrashVerifier: LastRunCrashVerifier by singleton {
-        LastRunCrashVerifier(crashMarker, initModule.logger)
-    }
-
-    override val crashService: CrashService by singleton {
+    private val crashServiceV1: CrashService by singleton {
```

Comment:
nit: could these be named `crashService` and `crashDataSource` respectively?

Reply:
Updated to `legacyCrashService` and `crashDataSource` because `crashService` is already used by the `CompositeCrashService` to keep references from other components.